tests/config/connection_config

```diff
@@ -0,0 +1,7 @@
+[connections.test]
```

Comment:
Does this file is meant to be used by other tests also or is it special only for the test of env vars substitution?
If it's the second case, I think we should think about more specific name to not mislead us and others in the future (I'd think that this is some common connection configuration used in tests).

Reply:
I forgot to mention that I'm talking about the filename, not the section name.

Reply:
For now only one test is using it, so changed name to more specific